scipy/stats/tests/test_stats.py

```diff
@@ -2758,80 +2758,84 @@ class TestsPercentileOfScore:
     def f(self, *args, **kwargs):
         return stats.percentileofscore(*args, **kwargs)
 
-    @pytest.mark.parametrize("kind, result", [("rank",   40),
-                                              ("mean",   35),
+    @pytest.mark.parametrize("kind, result", [("rank", 40),
+                                              ("mean", 35),
                                               ("strict", 30),
-                                              ("weak",   40)])
+                                              ("weak", 40)])
     def test_unique(self, kind, result):
-        assert_equal(self.f([1,2,3,4,5,6,7,8,9,10], 4, kind=kind), result)
+        assert_equal(self.f([1, 2, 3, 4, 5, 6, 7, 8, 9, 10], 4, kind=kind),  result)
```

Comment:
Great thanks! Just this one is extra.
```suggestion
        assert_equal(self.f([1, 2, 3, 4, 5, 6, 7, 8, 9, 10], 4, kind=kind), result)
```